server.js

```diff
@@ -0,0 +1,6 @@
+var express = require('express');
```

Comment:
consider using `http-server` or `lite-server` to serve the `www` folder. This will allow us to remove the dependency for `express` and remove `server.js`.